pkg/mcs/tso/server/server.go

```diff
@@ -290,12 +615,20 @@ func CreateServerWrapper(cmd *cobra.Command, args []string) {
 	log.Info("TSO Config", zap.Reflect("config", cfg))
 
 	grpcprometheus.EnableHandlingTimeHistogram()
-
 	metricutil.Push(&cfg.Metric)
 
-	// TODO: Create the server
+	rand.Seed(time.Now().UnixNano())
 	ctx, cancel := context.WithCancel(context.Background())
-	svr := &Server{}
+	svr := &Server{
+		DiagnosticsServer: sysutil.NewDiagnosticsServer(cfg.Log.File.Filename),
```

Comment:
Do we also need to do something like `diagnosticspb.RegisterDiagnosticsServer(gs, s)` during `(*tso.Server).startGRPCServer`?

Reply:
good catch! added